src/metabase/query_processor.clj

```diff
@@ -441,6 +430,31 @@
           (assert (every? u/string-or-keyword? (:columns <>))))))))
 
 
+(defn- run-query
+  "The end of the QP middleware which actually executes the query on the driver.
+
+   If this is an MBQL query then we first call `mbql->native` which builds a database dependent form for execution and
+   then we pass that form into the `execute-query` function for final execution.
+
+   If the query is already a NATIVE query then we simply pass it through to `execute-query` unmodified."
```

Comment:
Should probably just write this as `*native*` instead of NATIVE. Elsewhere we use uppercase as a convention to denote params, and Markdown already has some pretty good ways to emphasize something.